ratelimit/examples_test.go

```diff
@@ -0,0 +1,26 @@
+package ratelimit_test
+
+import (
+	"time"
+
+	"github.com/ceshihao/ratelimiter/tokenbucket"
```

Comment:
This is no longer in the project.

Tio do the example can we create a test rate limiter that meets the interface :+1:

Reply:
Do you mean to add a fake limiter (e.g. `allwaysPassLimiter`) in the example instead of the one based on `juju/ratelimit`?

Reply:
yep, lets just add a fake that returns true (or get creative) but one that is not exported ... dont really want the dep just for an example.

Reply:
Done